e2e/playwright/package.json

```diff
@@ -7,20 +7,20 @@
         "prettier": "prettier --write ."
     },
     "devDependencies": {
-        "@applitools/eyes-playwright": "1.12.2",
-        "@mattermost/client": "7.1.0",
-        "@mattermost/types": "7.1.0",
-        "@percy/cli": "1.10.0",
+        "@applitools/eyes-playwright": "1.12.7",
+        "@mattermost/client": "7.3.0",
+        "@mattermost/types": "7.3.0",
+        "@percy/cli": "1.11.0",
         "@percy/playwright": "1.0.4",
-        "@playwright/test": "1.25.1",
-        "@typescript-eslint/eslint-plugin": "5.35.1",
-        "@typescript-eslint/parser": "5.35.1",
+        "@playwright/test": "1.27.1",
+        "@typescript-eslint/eslint-plugin": "5.40.1",
+        "@typescript-eslint/parser": "5.40.1",
         "deepmerge": "4.2.2",
-        "eslint": "8.22.0",
+        "eslint": "8.25.0",
         "form-data": "4.0.0",
         "isomorphic-unfetch": "3.1.0",
         "prettier": "2.7.1",
-        "typescript": "4.7.4",
-        "uuid": "8.3.2"
+        "typescript": "4.8.4",
```

Comment:
it's awesome that e2e can be on latest TS, webapp upgrades to latest will be a major pain with 400 errors 🤯